pmm/pmm2-upgrade-tests.groovy

```diff
@@ -2,114 +2,52 @@ library changelog: false, identifier: 'lib@master', retriever: modernSCM([
-                echo "Upgrading pmm2-client";
-                sudo yum clean all
-                sudo yum makecache
-                sudo yum -y install pmm2-client
-                sudo yum -y update
-                sudo chmod 755 /srv/pmm-qa/pmm-tests/check_client_upgrade.sh
-                bash -xe /srv/pmm-qa/pmm-tests/check_client_upgrade.sh ${PMM_VERSION} ${PRE_POST}
-            '
-        """
-    }
-}
-
-void uploadAllureArtifacts() {
-    withCredentials([sshUserPrivateKey(credentialsId: 'aws-jenkins', keyFileVariable: 'KEY_PATH', passphraseVariable: '', usernameVariable: 'USER')]) {
-        sh """
-            scp -r -i "${KEY_PATH}" -o ConnectTimeout=1 -o StrictHostKeyChecking=no \
-                tests/output/allure aws-jenkins@${MONITORING_HOST}:/home/aws-jenkins/allure-reports
-        """
-    }
+    sh """
+        export PMM_VERSION=${PMM_VERSION}
+        echo "Upgrading pmm2-client";
+        sudo yum clean all
+        sudo yum makecache
+        sudo yum -y install pmm2-client
+        sleep 30
+        sudo chmod 755 /srv/pmm-qa/pmm-tests/check_client_upgrade.sh
+        bash -xe /srv/pmm-qa/pmm-tests/check_client_upgrade.sh ${PMM_VERSION} ${PRE_POST}
+    """
 }
 
 void fetchAgentLog(String CLIENT_VERSION) {
-     withCredentials([sshUserPrivateKey(credentialsId: 'aws-jenkins', keyFileVariable: 'KEY_PATH', passphraseVariable: '', usernameVariable: 'USER')]) {
-        sh """
-            ssh -i "${KEY_PATH}" -o ConnectTimeout=1 -o StrictHostKeyChecking=no ${USER}@${VM_IP} '
-                set -o errexit
-                set -o xtrace
-                export CLIENT_VERSION=${CLIENT_VERSION}
-                if [[ \$CLIENT_VERSION != http* ]]; then
-                    journalctl -u pmm-agent.service > /var/log/pmm-agent.log
-                    sudo chmod 777 /var/log/pmm-agent.log
-                fi
-                if [[ -e /var/log/pmm-agent.log ]]; then
-                    cp /var/log/pmm-agent.log .
-                fi
-            '
-            if [[ \$CLIENT_VERSION != http* ]]; then
-                scp -i "${KEY_PATH}" -o ConnectTimeout=1 -o StrictHostKeyChecking=no \
-                    ${USER}@${VM_IP}:pmm-agent.log \
-                    pmm-agent.log
-            fi
-        """
-    }
-    withCredentials([sshUserPrivateKey(credentialsId: 'aws-jenkins', keyFileVariable: 'KEY_PATH', passphraseVariable: '', usernameVariable: 'USER')]) {
-        sh """
-            if [[ \$CLIENT_VERSION == http* ]]; then
-                scp -i "${KEY_PATH}" -o ConnectTimeout=1 -o StrictHostKeyChecking=no \
-                    ${USER}@${VM_IP}:workspace/aws-staging-start/pmm-agent.log \
-                    pmm-agent.log
-            fi
-        """
-    }
+    sh """
+        export CLIENT_VERSION=${CLIENT_VERSION}
+        if [[ \$CLIENT_VERSION != http* ]]; then
+            journalctl -u pmm-agent.service > /var/log/pmm-agent.log
+            sudo chmod 777 /var/log/pmm-agent.log
```

Comment:
why do you need such insecure permissions?

Reply:
yes, changed it, thanks a lot for the input.